src/com/lftechnology/java/training/alina/jdbc/service/UserService.java

```diff
@@ -0,0 +1,324 @@
+        }
+    }
+
+    /**
+     * Gets employee info
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param userId
+     *            {@link Integer}
+     * @return employee {@link Employee} employee info
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static Employee setEmployeeInfo(Scanner scanner, int userId) {
+        Employee employee = new Employee();
+        employee.setFullname(UtilityService.getInputData(scanner, "Enter Fullname : "));
+        employee.setDepartment(UtilityService.getInputData(scanner, "Enter department : "));
+        employee.setAddress(UtilityService.getInputData(scanner, "Enter address : "));
+        Boolean roleStatus = false;
+        do {
+            String role = UtilityService.getInputData(scanner, "Enter role (user/admin): ");
+            roleStatus = EmployeeController.checkMatchedRole(scanner, roleStatus, role);
+            if (roleStatus) {
+                employee.setRole(role);
+            }
+        } while (!roleStatus);
+        employee.setUserId(userId);
+        employee.setCreatedAt(DateTimeService.getCurrentTimeStamp());
+        employee.setModifiedAt(DateTimeService.getCurrentTimeStamp());
+        return employee;
+    }
+
+    /**
+     * Gets role of normal user
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param result
+     *            {@link ResultSet}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private void getNormalUserRole(Scanner scanner, Employee employee) throws SQLException {
+        char choice = ' ';
+        while (choice != Constants.EMPLOYEE_EXIT) {
+            EmployeeView.displayEmployeeRoleMenu(employee);
+            choice = UtilityService.getSelectedMenu(scanner, "Select an option (a-d) : ");
+            getEmployeeRoleOptions(scanner, choice, employee);
+        }
+    }
+
+    /**
+     * Gets employee role menu options
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param choice
+     *            {@link Character}
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    private void getEmployeeRoleOptions(Scanner scanner, char choice, Employee employee) throws SQLException {
+        switch (choice) {
+        case 'a':
+            LOGGER.log(Level.INFO, "\n========================\nView Employee List : \n========================\n");
+            EmployeeController.getEmployeeList();
+            break;
+        case 'b':
+            LOGGER.log(Level.INFO, "\n========================\nSearch an Employee : \n========================\n");
+            EmployeeController.searchExistingEmployee(scanner);
+            return;
+        case 'c':
+            LOGGER.log(Level.INFO, "\n========================\nEdit own Information\n========================\n");
+            getEditInfo(scanner, employee);
+        case 'd':
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) End this switch case with an unconditional break, continue, return or throw statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS128)